src/test/java/SwitchDemo/SwitchDemo1.java

```diff
@@ -50,6 +50,7 @@ public static void main(String[] args){
         }
     }
 }
+//chirag code is completed.
```

Comment:
it is not completed. The assertion is not done. Can you please complete assertion.

Reply:
assertion is added